tools/deny.toml

```diff
@@ -65,25 +65,30 @@ multiple-versions = "deny"
 wildcards = "deny"
 
 skip = [
-    # num_cpus uses an old version of hermit-abi
-    { name = "hermit-abi", version = "=0.2.6" },
-    # tungstenite other crates use an old version of base64
-    { name = "base64", version = "=0.13.1" },
+    # several dependencies are using multiple versions of base64
+    { name = "base64" },
+    # several dependencies are using an old version of bitflags
+    { name = "bitflags", version = "=1.3" },
+    # several dependencies are using an old version of serde_yaml
+    { name = "serde_yaml", version = "=0.8" },
     # governor uses an old version of wasi
     { name = "wasi", version = "=0.10.2" },
+    # aws-sdk-rust is using an old version of fastrand
+    { name = "fastrand", version = "=1.9" },
+    # aws-sdk-rust is using an old version of rustls, hyper-rustls, and tokio-rustls
+    { name = "rustls", version = "=0.20" },
+    { name = "hyper-rustls", version = "=0.23" },
+    { name = "tokio-rustls", version = "=0.23" },
+    # kube-client uses an old version of redox_syscall
+    { name = "redox_syscall", version = "=0.2" },
```

Comment:
This looks like a lot, but it's because we're no longer skipping the entire `testsys-model` and `bottlerocket-types` trees.